Sources/Transaction/TransactionBuilder.swift

```diff
@@ -68,26 +105,27 @@ final class TransactionBuilder {
         rng: (@convention(c) (UnsafeMutableRawPointer?) -> UInt64)? = securityRNG,
         rngContext: Any? = nil
     ) -> Result<PendingSinglePayloadTransaction, TransactionBuilderError> {
-        positiveRemainingAmount(inputValues: inputs.map { $0.knownTxOut.value }, fee: fee)
-            .map { outputAmount in
-                PossibleTransaction([TransactionOutput(recipient, outputAmount)], nil)
-            }
-            .flatMap { possibleTransaction in
-                build(
-                    inputs: inputs,
-                    accountKey: accountKey,
-                    possibleTransaction: possibleTransaction,
-                    memoType: memoType,
-                    fee: fee,
-                    tombstoneBlockIndex: tombstoneBlockIndex,
-                    fogResolver: fogResolver,
-                    blockVersion: blockVersion,
-                    rng: rng,
-                    rngContext: rngContext
-                ).map { pendingTransaction in
-                    pendingTransaction.singlePayload
-                }
+        Math.positiveRemainingAmount(
+            inputValues: inputs.map { $0.knownTxOut.value },
+            fee: fee
+        ).map { outputAmount in
+            PossibleTransaction([TransactionOutput(recipient, outputAmount)], nil)
+        }.flatMap { possibleTransaction in
+            build(
+                inputs: inputs,
+                accountKey: accountKey,
+                possibleTransaction: possibleTransaction,
+                memoType: memoType,
+                fee: fee,
+                tombstoneBlockIndex: tombstoneBlockIndex,
+                fogResolver: fogResolver,
+                blockVersion: blockVersion,
+                rng: rng,
+                rngContext: rngContext
```

Comment:
It does look a bit strange, but for consistency...  (also same for the prior build in the file too):
```suggestion
                    inputs: inputs,
                    accountKey: accountKey,
                    possibleTransaction: possibleTransaction,
                    memoType: memoType,
                    fee: fee,
                    tombstoneBlockIndex: tombstoneBlockIndex,
                    fogResolver: fogResolver,
                    blockVersion: blockVersion,
                    rng: rng,
                    rngContext: rngContext)
```

Reply:
single indentation is the general rule